### PR TITLE
Tips to consider port forward as alt to ingress

### PR DESCRIPTION
**Pull request by johnsmartco**
**What this PR does**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [ ] Changes manually tested
- [ ] Chart versions updated (if necessary)
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [ ] CHANGELOG.md updated (not required for documentation PRs)
- [ ] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: docs/content/en/docs/topics/repair/_index.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ description: |
   Use the Reaper for Apache Cassandra (Reaper) web interface for repairs.
 ---
 
-Repairs are a critical anti-entropy operation in Apache Cassandra (Cassandra). In the past, there have been many custom solutions to manage them outside of your main Cassandra installation. K8ssandra provides the Reaper web interface (Reaper) that eliminates the need for a custom solution. Just like K8ssandra makes Cassandra setup easy, Reaper makes configuration of repairs even easier.
+Repairs are a critical anti-entropy operation in Apache Cassandra (Cassandra). In the past, there have been many custom solutions to manage them outside of your main Cassandra installation. K8ssandra provides the Reaper Web interface that eliminates the need for a custom solution. Just like K8ssandra makes Cassandra setup easy, Reaper makes configuration of repairs even easier.
```
Comment (jakerobb) [inline docs/content/en/docs/topics/repair/_index.md:9]: Do we need a `&reg;` here?

File: docs/content/en/docs/topics/repair/_index.md [hunk 2]
```diff
@@ -22,21 +22,30 @@ The requirement for your environment may vary considerably, however the general
...
 
 ![Reaper UI](reaper-main-ui.png)
 
 With the prerequisites satisfied the Reaper web interface should be available at the following address:
 
-http://REPAIR_DOMAIN/webui
+http://REAPER_DOMAIN/webui
 
-For example, to upgrade a previously installed `k8ssandra` that's running locally, where the `releaseName` used on the prior `helm install` command was `k8ssandra`:
+For example, to upgrade a previously installed `k8ssandra` that's running locally, where the `cluster-name` used on the prior `helm install` command was `k8ssandra`:
```
Comment (jakerobb) [inline docs/content/en/docs/topics/repair/_index.md:42]: The value you specify to `helm install` is releaseName, not clusterName. I'd avoid representing either with hyphenated values; using them in your release or cluster name leads to pod/service/etc names that are somewhat more difficult for humans to parse.

(clusterName defaults to be the same as releaseName if you don't specify something else)

File: docs/content/en/docs/topics/repair/_index.md [hunk 3]
```diff
@@ -62,7 +71,7 @@ prometheus-k8ssandra-kube-prometheus-stack-prometheus-0     2/2     Running
 
 ## What can I do in Reaper?
 
-To access Reaper, if you are running locally, navigate to [http://repair.localhost:8080/webui/](http://repair.localhost:8080/webui/).
+To access Reaper, if you are running locally, navigate to [http://localhost:8080/webui/](http://localhost:8080/webui/).
```
Comment (jakerobb) [inline docs/content/en/docs/topics/repair/_index.md:74]: "if you are running locally" is a bit confusing here; I'd move that up to the paragraph on line 48 and explain that they should choose something other than `localhost` if they're _not_ running locally, then add a note here to use whatever hostname you specified for `reaper.ingress.host`.

File: docs/content/en/docs/topics/repair/_index.md (hunk 1 above)
Comment (johnsmartco) [reply docs/content/en/docs/topics/repair/_index.md]: Yes, added. Thanks Jake.

File: docs/content/en/docs/topics/repair/_index.md (hunk 2 above)
Comment (johnsmartco) [reply docs/content/en/docs/topics/repair/_index.md]: Incorporated, thanks Jake.

File: docs/content/en/docs/topics/repair/_index.md (hunk 3 above)
Comment (johnsmartco) [reply docs/content/en/docs/topics/repair/_index.md]: Updated text with notes around lines 48 and 74, thanks Jake.

File: docs/content/en/docs/topics/repair/_index.md [hunk 4]
```diff
@@ -22,23 +22,34 @@ The requirement for your environment may vary considerably, however the general
...
+helm upgrade k8ssandra k8ssandra/k8ssandra --set reaper.ingress.enabled=true , reaper.ingress.host=localhost
+```
 
-Notice how in this example, the DNS host name is specified on the command line as `repair.localhost`.
+{{% alert title="Tip" color="success" %}}
+Notice how in the `helm upgrade` example above, with Ingress/Traefik access, the DNS hostname is specified on the command line as `localhost`. If you are not running locally, specify a hostname other than `localhost`. 
+{{% /alert %}}
 
-After a few minutes, check that the pods are running. Example:
+After about five or more minutes, check that the pods are running. Example:
```
Comment (jakerobb) [inline docs/content/en/docs/topics/repair/_index.md:52]: Are we interested in showing the user how to wait programmatically while monitoring progress?

`kubectl rollout status statefulset ${CLUSTERNAME}-${DATACENTER}-default-sts`

Assuming they've followed instructions without deviation, that would be:

`kubectl rollout status statefulset k8ssandra-dc1-default-sts`

If they run it _very_ quickly after `helm upgrade` or `helm install`, they may get an error that no statefulset with that name exists; in that case they should just run it again. The command will wait up to ten minutes (the timeout can be configured longer if desired) and will give a line of output as each pod in the set becomes ready.

**Review by jakerobb — APPROVED** [review]
LGTM. I added a tip about waiting for pods to be running.

File: docs/content/en/docs/topics/repair/_index.md (hunk 4 above)
Comment (johnsmartco) [reply docs/content/en/docs/topics/repair/_index.md:52]: Great tips, thanks Jake.

**Pull request merged**